src/osd/PG.cc

```diff
@@ -3994,6 +4009,16 @@ void PG::chunky_scrub(ThreadPool::TPHandle &handle)
 	  scrubber.reserved_peers.clear();
 	}
 
+	{
+	  hobject_t oid = Scrub::make_scrub_object(info.pgid.pgid);
```

Comment:
will do. and as will not encode the pg into the object name, as temp coll is per pg.